keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py

```diff
@@ -77,8 +92,8 @@ def test_saved_model(self):
         )
 
     @pytest.mark.large  # Saving is slow, so mark these large.
-    def test_saved_alias_model(self):
-        model = csp_darknet_backbone.CSPDarkNetLBackbone()
+    def test_saved_alias_model(self, save_format, filename):
```

Comment:
This change breaks CI, since this test is no longer parameterized

Reply:
Done.